core/fullHistory/proto/historyTransaction.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "fullHistory";
+option (gogoproto.stable_marshaler_all) = true;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+//HistoryTransaction is used to store information about a history transaction
+message HistoryTransaction {
+    uint32 RcvShardID  = 1;
+    uint32 SndShardID  = 2;
+    uint64 Round       = 3;
+    uint64 HeaderNonce = 4;
+    bytes  HeaderHash  = 5;
+	bytes  MbHash      = 6;
+	bytes  Status      = 7;
+}
```

Comment:
these fields are sufficient because on get transaction we do a merge between information from this structure and information that are saved in storage for a transaction